site/ja/docs/privacy-sandbox/attribution-reporting-changes-january-2022/index.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Attribution Reporting proposal: updates in January 2022'
+date: 2022-01-27
+updated: 2022-01-27
+authors:
+  - maudn
+---
+
+このアトリビューション
+レポートの提案には、コミュニティからのフィードバックに対応するため、API
+メカニズムの変更から新機能まで、さまざまな変更が反映されています。
+
+これらの変更についてフィードバックをお寄せください。さまざまなユースケースに対する解決策をオープンに議論するには、エコシステムからのアイデアが不可欠です。
+
+議論に参加するには、この投稿の「**公開ディスカッションに参加**」の下にあるリンクでコメントするか、新しいイシューを作成してください。[参加の詳細についてご確認ください](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-introduction/#participate)。
+
+## 変更履歴
+
+- 2022 年 2 月 7 日: [ヘッダートリガーのリダイレクト](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-changes-january-2022/#header-trigger-redirect)に関するセクションを追加しました。
+- 2022 年 1 月 27 日: 記事を最初に公開しました。
+
+## この投稿の対象者
+
+この投稿は次の方を対象としています。
+
+- この API についてしっかりと理解している方。たとえば、WICG リポジトリでの議論を見守っていたり参加したりしていて、2022 年 1 月の提案に加えられた変更を把握したい方。
+- デモや本番環境のテストで Attribution Reporting API を使用している方。
+
+この API を初めて検討する場合や、まだ試してみていない場合は、まず、[API
+の概要](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-introduction/)をご覧ください。
```

Comment:
Links to developer.chrome.com are relative. Please remove `https://developer.chrome.com` from all links in the page.